polling_stations/apps/pollingstations/models.py

```diff
@@ -35,6 +35,13 @@ def __unicode__(self):
         return "%s (%s)" % (name, self.council)
 
 
+class VisibilityChoices(models.TextChoices):
+    # Visible on website and in API.
+    PUBLISHED = "PUBLISHED", "Published"
+    # Only visible to upload user and admin. I.e. not on website or API.
+    UNPUBLISHED = ("UNPUBLISHED", "Unpublished")
```

Comment:
[formatting] is this a Ruff thing? Looks odd having braces for one and not the other!